src/main/java/lotto/domain/LottoRank.java

```diff
@@ -0,0 +1,36 @@
+package lotto.domain;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public enum LottoRank {
+	FOURTH(3, 5000),
+	THIRD(4, 50000),
+	SECOND(5, 1500000),
+	FIRST(6, 2000000000);
```

Comment:
언더스코어 표기법을 통해 가독성을 높일 수 있어요 :)
```suggestion
	FIRST(6, 2_000_000_000);
```